core/amber/src/main/scala/edu/uci/ics/texera/web/resource/auth/GoogleClientIdSecret.java

```diff
@@ -0,0 +1,6 @@
+package edu.uci.ics.texera.web.resource.auth;
+
+public class GoogleClientIdSecret {
+    public static final String CLIENT_ID = "";
+    public static final String CLIENT_SECRETE = "";
+}
```

Comment:
those can be read from config, you can checkout `web.conf` file, we have already had some google-related configs there, please append to it.

Reply:
done